stories/Hits.stories.js

```diff
@@ -1,7 +1,15 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
-import { Hits, Highlight, Panel, Snippet } from 'react-instantsearch-dom';
+import { action } from '@storybook/addon-actions';
+import {
+  Hits,
+  Highlight,
+  Panel,
+  Snippet,
+  Configure,
+} from 'react-instantsearch-dom';
+import { connectHitsInsights } from 'react-instantsearch-core';
```

Comment:
The users don't have to use the `core`, we should also export the connector from the [DOM](https://github.com/algolia/react-instantsearch/blob/master/packages/react-instantsearch-dom/src/index.js) package also.

Reply:
fixed here: 2c84002